lib/api.ts

```diff
@@ -0,0 +1,243 @@
+import {
+  AuthorizationType,
+  FieldLogLevel,
+  GraphqlApi,
+  MappingTemplate,
+  SchemaFile,
+} from "@aws-cdk/aws-appsync-alpha";
+import { AttributeType, BillingMode, Table } from "aws-cdk-lib/aws-dynamodb";
+import { Construct } from "constructs";
+import { join } from "path";
+import { SFNPutNodes } from "./sfn-put-node";
+
+interface CompendiumApiProps {}
+
+export class CompendiumAPI extends Construct {
+  constructor(scope: Construct, id: string, props: CompendiumApiProps) {
+    super(scope, id);
+
+    const dynamoTable = new Table(this, `${id}Table`, {
+      tableName: id,
```

Comment:
Don't we want to use some static table name instead?

Reply:
agreed, fixed